src/libslic3r/PrintBase.cpp

```diff
@@ -101,6 +101,11 @@ std::string PrintBase::output_filepath(const std::string &path, const std::strin
     return path;
 }
 
+std::string PrintBase::output_filepath_for_upload() const
```

Comment:
Why we need a new function here? what's the difference with output_filepath?

Reply:
a tired brain saw function, so it made a function XD. just fixed it with a new commit